src/docs/asciidoc/Analytics.asciidoc

```diff
@@ -331,301 +331,11 @@ datasets$testset[,1:4][1,]
 [1] versicolor
 ----
 
-[[counters-and-gauges]]
 === Counters and Gauges
```

Comment:
maybe we should slightly change this section title, to avoid autogenerated id issues